src/Commands/ResizeImagesCommand.php

```diff
@@ -13,7 +15,25 @@ class ResizeImagesCommand extends Command
 
     public function handle(): int
     {
-        ResizeImagesJob::dispatch();
+        $verboseLevel = $this->getOutput()->getVerbosity();
+
+        if ($verboseLevel > 32) {
```

Comment:
Probably better to use: https://laravel.com/api/9.x/Illuminate/Console/OutputStyle.html e.g. $this->getOutput()->isVeryVerbose()